components/dropdown/basic/src/index.js

```diff
@@ -70,14 +71,18 @@ class DropdownBasic extends Component {
   /**
    * Function rendering a simple list item link.
    */
-  _renderLink = ({text, url, target}, index) => {
+  _renderLink = ({onClick, target, text, url}, index) => {
     const Link = this.props.linkFactory
 
     return (
       <li key={index} className="sui-DropdownBasicMenu-listItem">
         <Link
           href={url}
           className="sui-DropdownBasicMenu-listLink"
+          onClick={e => {
```

Comment:
Don't create inline arrow functions to handle events. They result in more memory consuming.

Reply:
👍